File: beginner/1019/TimeConversion.java
// challenge link - https://judge.beecrowd.com/en/problems/view/1019

import java.io.IOException;
import java.util.Scanner;

public class TimeConversion {
	public static void main(String[] args) throws IOException {
		Scanner reader = new Scanner(System.in);
		int totalSeconds = reader.nextInt();
		reader.close();

		int hours = totalSeconds / 3600;
		totalSeconds = totalSeconds % 3600;
		int minutes = totalSeconds / 60;
		int seconds = totalSeconds % 60;

		System.out.println(hours + ":" + minutes + ":" + seconds);
	}
}

File: beginner/1019/time_conversion.c
// challenge link - https://judge.beecrowd.com/en/problems/view/1019

#include <stdio.h>

int main(void) {
	int total_seconds = 0;
	scanf("%d", &total_seconds);

	int units[] = {3600, 60}; // hours, minutes
	int num_of_units = 2;
	int unit = 0;
	int i = 0;
	
	while (i < num_of_units) {
		unit = total_seconds / units[i];
		total_seconds = total_seconds % units[i];
		printf("%d:", unit);
		i++;
	}

	printf("%d\n", total_seconds);
	return (0);
}
